rfcs/040-tei_adapter/README.md

```diff
@@ -0,0 +1,109 @@
+# TEI Adapter
+
+## What is TEI
+
+TEI is an XML standard to represent any source in any script or language. 
+It's used at Wellcome to describe manuscripts in non latin languages.
+The TEI files are in a [GitHUb Repository](https://github.com/wellcomecollection/wellcome-collection-tei). At the 
+time of writing in April 2021 there are 622 TEI files, but the collection of manuscripts is much larger, around 10k.
+
+Currently, all information stored in TEI files is not surfaced anywhere except the GitHub repository. 
+Many of the TEI files have a dummy Sierra record that they link to, but not all
+## How we do it
+### Do we need an adapter?
+
+We need to be able to propagate updates to TEI files as they happen through to the API, and we also need to be able to
+reindex when we make changes in the pipeline. 
+
+GitHub is a relatively reliable and low latency source, so we could in theory just send the changes to a TEI transformer 
+and do a reindex by cloning the repository and/or parsing the tree of the TEI files. 
+There are some disadvantages to this approach:
+- We need a "special" reindexer for TEI.
+- We can't keep track of deleted files. If something is deleted, the change will go through the pipeline once, 
+  but there's no way of "remembering" it.
```

Comment:
I think I get what you're saying, but it's almost the opposite problem – we'll remember it forever, we can't forget it.

Reply:
You're right. What I meant to say is that there's no way of remembering that it used to exist. I'll rephrase, thanks